Moo/play/lib.py

```diff
@@ -354,8 +354,9 @@ def index(config, sort=None):
     return sorted(albums, key=sort_key, reverse=True)
 
 
-def info(track, metadata):
+def album(track, metadata):
     '''
+    Changed def info to def album()
```

Comment:
You changed lib.info() to lib.album() which is what I asked for. However, now I see that having albums() and album() is confusing. Let's change lib.info() to lib.album_info() instead.